jazz_apigee-proxy-aws/components/config.js

```diff
@@ -0,0 +1,60 @@
+// =========================================================================
+// Copyright © 2017 T-Mobile USA, Inc.
+// 
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// =========================================================================
+
+/**
+	Nodejs Template Project
+  @module: config.js
+  @description: Defines variables/functions to retrieve environment related data
+	@author:
```

Comment:
Spacing is off.

Reply:
@svidhani Can you make sure whatever editor you're using has Editorconfig support installed/enabled? That should keep everything consistent.